src/actions/timelineActions.js

```diff
@@ -0,0 +1,90 @@
+import axios from "axios";
+import * as types from "./actionTypes";
+import * as api from "../util/api.js"
+import store from "../store"
+
+function receiveTimeline(judgments, raceGroups, stage, gpsData) {
+    const jsonData = JSON.parse(JSON.stringify(gpsData));
+    const speedLeadGroup = jsonData.cars.find(function (obj) { return obj.name === "verkehrsspitze"; }).data.speed * 3.6;
+    const distanceLeadGroup = jsonData.cars.find(function (obj) { return obj.name === "verkehrsspitze"; }).data.distance.toFixed(1);
+    let elements = [];
+    judgments.map(judgment => {elements.push({distance : judgment.distance, text : judgment.name})});
+    raceGroups.map(raceGroup => { elements.push({distance : distanceLeadGroup - raceGroup.actualGapTime * speedLeadGroup, text : raceGroup.raceGroupType})});
+    elements.sort((a,b) => b.distance - a.distance);
+    let elems = [];
+    let lastElement = null;
+    let size = elements.length;
+    elements.map((elem, i) => {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add a "return" statement to this callback. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3796)